bindings/py/nupic/bindings/check.py

```diff
@@ -0,0 +1,42 @@
+# ----------------------------------------------------------------------
+# Numenta Platform for Intelligent Computing (NuPIC)
+# Copyright (C) 2015, Numenta, Inc.  Unless you have an agreement
+# with Numenta, Inc., for a separate license for this software code, the
+# following terms and conditions apply:
+#
+# This program is free software: you can redistribute it and/or modify
+# it under the terms of the GNU Affero Public License version 3 as
+# published by the Free Software Foundation.
+#
+# This program is distributed in the hope that it will be useful,
+# but WITHOUT ANY WARRANTY; without even the implied warranty of
+# MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE.
+# See the GNU Affero Public License for more details.
+#
+# You should have received a copy of the GNU Affero Public License
+# along with this program.  If not, see http://www.gnu.org/licenses.
+#
+# http://numenta.org/licenses/
+# ----------------------------------------------------------------------
+
+
+
+def checkImportBindings():
+  try:
+    import nupic.bindings.math
+    import nupic.bindings.algorithms
+    import nupic.bindings.regions
+    import nupic.bindings.engine_internal
+  except:
+    return False
+  else:
+    return True
+
+
+
+def printImportBindingsMessage():
+  if checkImportBindings():
```

Comment:
I would split `checkImportBindings` into two functions:

1. One that imports "nupic.bindings". Try this first and if it fails print something like "Could not import nupic.bindings. It must be installed before use."
2. One that imports the three extensions (algorithms, engine_internal, and math). If this fails then print something like "Could not import C extensions for nupic.bindings. Make sure that the package was properly installed."